setup.py

```diff
@@ -63,10 +63,10 @@ def run(self):
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
```

Comment:
Unrelated to this py3.5 change but just confirming that py3.9 is being tested:

https://github.com/jupyter/jupyter_client/blob/018d8da0b947865d8217b459197bcc4f13e04666/.github/workflows/main.yml#L41

👍